core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -149,7 +149,19 @@ var TopicsAndSkillsDashboardPage = function() {
     topicNameField.sendKeys(title);
     abbreviatedTopicNameField.sendKeys(abbreviatedName);
     confirmTopicCreationButton.click();
-    waitFor.pageToFullyLoad();
+
+    waitFor.newTabToBeCreated('Creating topic takes too long');
+    return browser.getAllWindowHandles().then(function(handles) {
+      var firstTabHandle = handles[0];
+      var secondTabHandle = handles[1];
+      return browser.switchTo().window(secondTabHandle).then(function() {
+        waitFor.pageToFullyLoad();
```

Comment:
Why do we need to wait for the page to load if we're just going to close it? This seems like something we want to call after we've switched to the new tab

Reply:
Done